hw03_carnesa.py

```diff
@@ -0,0 +1,326 @@
+    clw.forward(120)
+    clw.left(70)
+    clw.forward(130)
+    clw.right(120)
+    clw.circle(30, 50)
+    clw.right(90)
+    clw.forward(90)
+    clw.left(90)
+    clw.forward(70)
+    clw.left(90)
+    clw.forward(80)
+    clw.right(100)
+    clw.circle(20, 50)
+    clw.right(90)
+    clw.forward(50)
+    clw.left(90)
+    clw.forward(50)
+    clw.left(90)
+    clw.forward(50)
+    clw.right(70)
+    clw.circle(100, 40)
+    clw.left(50)
+    clw.circle(70, 50)
+    clw.circle(1, 40)
+    clw.left(5)
+    clw.circle(-80, 140)
+    clw.forward(160)
+    clw.left(165)
+    clw.forward(130)
+    clw.right(80)
+    clw.forward(30)
+    clw.end_fill()
+    clw.hideturtle()
+
+
+def crest_right_wing(crw):
+    """
+    Doc string for crest_right_wing function. Calls the crescent_right turtle and draws the associated image.
+    """
+    # Mirrored to left wing on the opposite side of the Triforce.
+    crw.right(-120)
+    crw.forward(-30)
+    crw.pendown()
+    crw.begin_fill()
+    crw.right(-90)
+    crw.forward(-220)
+    crw.left(-120)
+    crw.forward(-120)
+    crw.left(-70)
+    crw.forward(-200)
+    crw.right(-100)
+    crw.circle(50, -50)
+    crw.right(-100)
+    crw.forward(-190)
+    crw.left(-130)
+    crw.forward(-120)
+    crw.left(-70)
+    crw.forward(-130)
+    crw.right(-120)
+    crw.circle(30, -50)
+    crw.right(-90)
+    crw.forward(-90)
+    crw.left(-90)
+    crw.forward(-70)
+    crw.left(-90)
+    crw.forward(-80)
+    crw.right(-100)
+    crw.circle(20, -50)
+    crw.right(-90)
+    crw.forward(-50)
+    crw.left(-90)
+    crw.forward(-50)
+    crw.left(-90)
+    crw.forward(-50)
+    crw.right(-70)
+    crw.circle(100, -40)
+    crw.left(-50)
+    crw.circle(70, -50)
+    crw.circle(1, -40)
+    crw.left(-5)
+    crw.circle(-80, -140)
+    crw.forward(-160)
+    crw.left(-165)
+    crw.forward(-130)
+    crw.right(-80)
+    crw.forward(-30)
+    crw.end_fill()
+    crw.hideturtle()
+
+
+def crest_crystal(cry):
+    """
+    Doc string for crest_crystal function. Calls the crest_cry turtle and draws the associated image.
+    """
+    cry.left(68)
+    # Triangle
+    cry.pendown()
+    cry.begin_fill()
+    cry.forward(150)
+    cry.right(135)
+    cry.forward(150)
+    cry.right(113)
+    cry.forward(117)
+    cry.end_fill()
+    # Rhombus
+    cry.begin_fill()
+    cry.backward(57)
+    cry.left(115)
+    cry.forward(100)
+    cry.right(47)
+    cry.forward(100)
+    cry.right(135)
+    cry.forward(100)
+    cry.right(47)
+    cry.forward(100)
+    cry.left(114)
+    cry.end_fill()
+    # Arc Right
+    cry.fillcolor("#C3C91B")
+    cry.begin_fill()
+    cry.backward(50)
+    cry.right(7)
+    cry.circle(25, 111)
+    cry.forward(60)
+    cry.end_fill()
+    # Arc Left
+    cry.begin_fill()
+    cry.right(168)
+    cry.forward(99)
+    cry.left(64)
+    cry.forward(46)
+    cry.circle(25, -100)
+    cry.backward(58)
+    cry.right(13)
+    cry.forward(90)
+    cry.end_fill()
+    cry.hideturtle()
+    # ...
```

Comment:
Remove help text from the template.

Reply:
There are multiple of these, check for them.

Reply:
@sheggen The additional text has been removed.